examples/rust_lang_tester/lang_tests/unknown_var.rs

```diff
@@ -0,0 +1,11 @@
+// [Compiler]
+// status = "error"
```

Comment:
Are the only possibilities for `status` `"error"` and `"success"`? In which case, would it make more sense for this to be `success = true | false`?

Reply:
This is one of those things I'm a bit unsure about. We *could* make this test for Unix error returns (which would be integers), but I reckon 99% of the time you just care about "succeeded" or "errored". But, yes, I tend to agree that "true", "false" is better.

Reply:
We've just had an in-person conversation:

If we were going to be super-anal, we should be checking the exit status against an integer. Laurie isn't keen on doing that now, but if we leave the keyword as "status", then that allow us to at-least implement later.

Put differently, if we commit to `success=true/false`, we can't later add `success=6`, which makes little sense.

So I'd say, leave it as it is.

Reply:
Yes that sounds reasonable to me